templates/test/comments.html.ep

```diff
@@ -0,0 +1,37 @@
+% if (!$job->comments->all) {
+    <i>No comments</i>
+% } else {
```

Comment:
I was thinking, is it even necessary to have 'No comments' there? I looked how it looks deployed and as is it seems out of place and it adds one more resultset->array conversion (which I don't exactly know how costly is, so maybe fine). There is already 0 in comments header if none is there.